demo/src/app/components/modal/demos/customclass/modal-customclass.ts

```diff
@@ -0,0 +1,33 @@
+import {Component, ViewEncapsulation} from '@angular/core';
+
+import {NgbModal, ModalDismissReasons} from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+  selector: 'ngbd-modal-customclass',
+  templateUrl: './modal-customclass.html',
+  encapsulation: ViewEncapsulation.None,
+  styleUrls: [ './modal-customclass.css' ]
+})
+export class NgbdModalCustomClass {
+  closeResult: string;
+
+  constructor(private modalService: NgbModal) {}
+
+  open(content) {
+    this.modalService.open(content, { windowClass: 'dark-modal' }).result.then((result) => {
+      this.closeResult = `Closed with: ${result}`;
```

Comment:
I don't think we need to repeat all the demo here. Just `this.modalService.open(...)` should be enough, IMO